patches/latency_info.patch

```diff
@@ -0,0 +1,65 @@
+diff --git a/ui/latency/BUILD.gn b/ui/latency/BUILD.gn
+index b024237..88318d1 100644
+--- a/ui/latency/BUILD.gn
++++ b/ui/latency/BUILD.gn
+@@ -17,6 +17,8 @@ source_set("latency") {
+     "//base",
+     "//ui/gfx",
+   ]
++
++  defines = [ "UI_LATENCY_IMPLEMENTATION" ]
+ }
+ 
+ test("latency_unittests") {
+diff --git a/ui/latency/latency_info.h b/ui/latency/latency_info.h
+index b358fba..5aa05a4 100644
+--- a/ui/latency/latency_info.h
++++ b/ui/latency/latency_info.h
+@@ -14,6 +14,7 @@
+ 
+ #include "base/containers/small_map.h"
+ #include "base/time/time.h"
++#include "ui/latency/ui_latency_export.h"
+ #include "ui/gfx/geometry/point_f.h"
+ 
+ #if !defined(OS_IOS)
+@@ -113,7 +114,7 @@ enum SourceEventType {
+   SOURCE_EVENT_TYPE_LAST = OTHER,
+ };
+ 
+-class LatencyInfo {
++class UI_LATENCY_EXPORT LatencyInfo {
+  public:
+   struct LatencyComponent {
+     // Nondecreasing number that can be used to determine what events happened
+diff --git a/ui/latency/ui_latency_export.h b/ui/latency/ui_latency_export.h
+new file mode 100644
```

Comment:
The problem with this approach is that the patch will be successfully applied only for the first time. If you run `./script/update` once more, `ui_latency_export.h` will already be present from the first run and `git apply` will fail.